src/app/pages/MediaArticlePage/MediaArticlePage.tsx

```diff
@@ -112,18 +112,35 @@ const MediaArticlePage = ({ pageData }: MediaArticlePageProps) => {
     pageData,
   );
 
+  // ATI
+  const {
+    metadata: { atiAnalytics, type },
+  } = pageData;
+
+  const isMap = type === 'MAP';
+
   const componentsToRender = {
     fauxHeadline,
     visuallyHiddenHeadline,
     headline: headings,
     subheadline: headings,
     audio: (props: ComponentToRenderProps) => (
-      <div css={styles.mediaPlayer}>
+      <div
+        css={[
+          `padding-top:${GEL_SPACING_TRPL}`,
```

Comment:
Rather than using the psammead styles, you should be able to use the Theme directly in these `css` props, like: https://github.com/bbc/simorgh/blob/51a5e3dc1a32c8f942da1298b73a1684d50f6c5f/src/app/components/Curation/HierarchicalGrid/index.tsx#L82